.travis.yml

```diff
@@ -2,7 +2,11 @@ services:
 - docker
 
 addons:
-  postgresql: "9.6"
+  postgresql: "11"
```

Comment:
Huh -- pretty sure that didn't exist when I first tried, which is why I went with the Docker approach. This makes it easier.